cmd/server/main.go

```
@@ -0,0 +1,63 @@
+func main() {
+	r := gin.Default()
+
+	godotenv.Load()
+	endpoint := os.Getenv("DYNAMO_ENDPOINT")
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "ap-northeast-1"
+	}
+	cfg, err := config.LoadDefaultConfig(context.Background(),
+		config.WithRegion(region),
+		config.WithEndpointResolver(
```

golangci-lint (1.64.8)

SA1019: config.WithEndpointResolver is deprecated: The global endpoint resolution interface is deprecated. The API for endpoint resolution is now unique to each service and is set via the EndpointResolverV2 field on service client options. Use of WithEndpointResolver or WithEndpointResolverWithOptions will prevent you from using any endpoint-related service features released after the introduction of EndpointResolverV2. You may also encounter broken or unexpected behavior when using the old global interface with services that use many endpoint-related customizations such as S3. 

(staticcheck)